hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/sink/compact/HoodieFlinkCompactor.java

```diff
@@ -211,28 +214,36 @@ private void compact() throws Exception {
 
       // checks the compaction plan and do compaction.
       if (cfg.schedule) {
-        Option<String> compactionInstantTimeOption = CompactionUtil.getCompactionInstantTime(metaClient);
+        Option<String> compactionInstantTimeOption = CompactionUtil.getCompactionInstantTime(
+            metaClient);
         if (compactionInstantTimeOption.isPresent()) {
-          boolean scheduled = writeClient.scheduleCompactionAtInstant(compactionInstantTimeOption.get(), Option.empty());
+          boolean scheduled = writeClient.scheduleCompactionAtInstant(
+              compactionInstantTimeOption.get(), Option.empty());
```

Comment:
Can we revert these unnecessary changes ?

Reply:
Sure.